# Empty
---
# Mesh response
TextureMesh texture_mesh
